src/pages/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "../LandingPage/landingPage.css";
+
+export function LandingPage({ onButtonClick }) {
```

Comment:
I think that logic for handling a component's events should live inside of the component.

For that reason, I think that you should move the event handler function for the button press into this component.

You will end up having to pass down the dispatch instead of a function, and that is okay because the dispatch is supposed to live at a higher level.

Reply:
I agree. Thanks for the criticism. Changes reflected in the code; I have submitted for re-review.

Reply:
Looks great.  Amazing job!

I found one other little bug that I pushed up.  I'm ready to approve.